src/Razor/Razor/src/TagHelpers/NullHtmlEncoder.cs

```diff
@@ -11,12 +12,12 @@ namespace Microsoft.AspNetCore.Razor.TagHelpers
     /// A <see cref="HtmlEncoder"/> that does not encode. Should not be used when writing directly to a response
     /// expected to contain valid HTML.
     /// </summary>
-    public class NullHtmlEncoder : HtmlEncoder
+    public sealed class NullHtmlEncoder : HtmlEncoder
     {
         /// <summary>
         /// Initializes a <see cref="NullHtmlEncoder"/> instance.
         /// </summary>
-        protected NullHtmlEncoder()
+        private NullHtmlEncoder()
```

Comment:
breaking change

Reply:
I assume it's not very important as it should not make sense to inherit from something that does nothing, also because we don't expose this type in public surface apis, and because one can easily inherit from the same base class. Is that an issue in this case? Or just a matter of Announcement?

Reply:
I agree but we need to leave a paper trail so that we can easily query them. I'm not sure this is worth an announcement but maybe it is?

Reply:
This is resolved now - I've filed an issue and marked as necessary